README.md

```diff
@@ -1,73 +1,57 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
+# to do List
+> To do list is a simple CRUD application that allows you to add, delete, update tasks that you are supposed to do.
 
-# Project Name
-
-> Description the project.
-
-
+![screenshot](./one.png)
```

Comment:
Great Job. Your Readme is descriptive and well organized. But screenshot in your Readme is not appearing. It would be wonderful if you fixed it as it will maintain the structure of the Readme.
![Screenshot from 2022-02-22 00-20-59](https://user-images.githubusercontent.com/60083437/155009050-0063dc20-d0d5-4210-b8e9-0e9973357e44.png)

Reply:
Thank you for your review @Tufahel I have implemented the changes